changelogs/CHANGELOG.rst

```diff
@@ -3,3 +3,12 @@ Ansible Network Collection Release Notes
 ==========================================
 
 .. contents:: Topics
+
+
+v1.0.0
+======
+
+Bugfixes
+--------
+
+- test
```

Comment:
Did you mean to leave this here? Maybe a release_summary section would be better?

Reply:
agreed!! updated the same.